lib/utils.pm

```diff
@@ -1120,4 +1121,54 @@ sub prepare_ssh_localhost_key_login {
     }
 }
 
+=head2 script_run_interactive
+For interactive command, input strings or keys according to the prompt message
+in the run time. Pass arrayref which contains the prompt message to be matched
+(regex) and the answers
```

Comment:
Could you please add an example of the usage as you use e.g. hard-coded `$endmark`, which one can observe on serial during the investigation.

Reply:
I added an example for calling this subroutine, as well as a explanation for the mark of command finishing.